src/workers/parser/tests/__snapshots__/getSymbols.spec.js.snap

```diff
@@ -755,10 +755,13 @@ hasJsx: false"
 
 exports[`Parser.getSymbols flow 1`] = `
 "functions:
-[(2, 2), (4, 3)]   renderHello(name) App
+[(2, 2), (4, 3)]   renderHello(name, action, ) App
 
 variables:
 [(2, 14), (2, 26)]   name
+[(2, 28), (2, 47)]   action
+[(2, 49), (2, 65)]
```

Comment:
should we exclude this?

Reply:
Okay, so i think this is a different bug with object keys `foo( { todos }: Props) {...}`

lets move forward and fix this up later